src/ui/czi-ordered-list.css

```diff
@@ -1,18 +1,31 @@
+html {
+  /* This sets up the initial counter values to `0`. */
+  counter-reset:
+    czi-counter-0 czi-counter-1 czi-counter-2 czi-counter-3
+    czi-counter-4 czi-counter-5 czi-counter-6 czi-counter-7;
+}
+
 /* For MS-Edge 16+, Chrome, Safari, Firefox */
 .ProseMirror ol {
   --czi-counter-name: czi-counter-0;
   --czi-list-style-type: decimal;
   --czi-counter-reset: 0;
+
+  counter-reset: none;
 }
 
 .ProseMirror ol > li {
   counter-increment: var(--czi-counter-name);
 }
 
-.ProseMirror ol > li:first-child {
+.ProseMirror ol:not([data-counter-reset='none']) {
```

Comment:
seems like we can directly handle these in `toDOM` since we're already setting styles there.

Reply:
That's right.

But I also want to keep all most of the css counters logics  at the same file `src/ui/czi-ordered-list.css` as possible as I could cause this CSS file gives more context about how the counter variables are being used.